how-to-docs/amazon-resources.md

```diff
@@ -0,0 +1,27 @@
+# How to get Amazon resources in the UK cluster
+
+Amazon resources in the UK Cluster can be requested by creating an issue on the [application-container-platform bau board](https://github.com/UKHomeOffice/application-container-platform-bau/issues)
+
+## RDS Instances
```

Comment:
For these requests we'd probably also want them to confirm the cluster / environment that the resource should be located with.